app/templates/Gruntfile.js

```diff
@@ -128,13 +128,6 @@ module.exports = function (grunt) {
           '<%%= yeoman.client %>/{app,components}/**/*.spec.{coffee,litcoffee,coffee.md}'
         ],
         tasks: ['karma']
-      },<% } %><% if(filters.babel) { %>
-      babel: {
-        files: [
-          '<%%= yeoman.client %>/{app,components}/**/*.js',
-          '!<%%= yeoman.client %>/{app,components}/**/*.spec.js'
-        ],
-        tasks: ['babel']
```

Comment:
We're already watching the exact same files in the `watch.injectJS` config; so we can just add our task there.